vector/v.in.ogr/main.c

```diff
@@ -1175,13 +1175,15 @@ int main(int argc, char *argv[])
 			      Ogr_fieldname, OFTIntegerListlength);
 		}
 		else {
-		    G_warning(_("Column type (Ogr_ftype: %d) not supported (Ogr_fieldname: %s)"),
-			      Ogr_ftype, Ogr_fieldname);
-		    buf[0] = 0;
-		    col_info[i_out].type = G_store(buf);
-		}
-		G_free(Ogr_fieldname);
-	    }
+                /* handle columns of unsupported data type */
+                G_warning(_("Column <%s> is of unsupported \"%s\" data type, "
+                            "it is omitted from import."),
```

Comment:
The message can be simpler something like:

```
"Column <%s> of unsupported data type \"%s\" is omitted from import"
```

Reply:
Warning simplified as suggested in [ba86d84](https://github.com/OSGeo/grass/pull/2630/commits/ba86d84c01556fa6f0107e6bf9bf0b266ea0efd7)